docroot/themes/humsci/humsci_basic/src/scss/components/_vertical-link-card.scss

```diff
@@ -0,0 +1,112 @@
+.vertical-link-card {
+  position: relative;
+  // The margin-top matches the margin-top on both the title and image to prevent
+  // overlap issues.
+  margin-top: hb-calculate-rems(24px);
+  // The margin-left matches the negative margin-left on the title to prevent
+  // overlap issues.
+  margin-left: hb-calculate-rems(31px);
+  padding-bottom: hb-calculate-rems(28px);
+
+  @include hb-colorful {
+    color: $hb-color--white;
+
+    // The gradient adds a 20px hight white block at the top of the linked card
+    // so that the title looks like it has a negative margin on the top when
+    // there isn't an image.
+    background-image: linear-gradient(to bottom, $hb-color--white, $hb-color--white hb-calculate-rems(20px), darken(hb-colorful-variation(tertiary), 30%) hb-calculate-rems(20px));
+    // The gradient looks intimidating but it's only because of all the mixins
+    // we have to use. If we get rid of those it looks more like this:
+    // linear-gradient(to bottom, white, white 20px, colorful-color 20px);
+  }
+
+  &__img {
+    margin-top: hb-calculate-rems(-24px);
+    overflow: hidden;
+
+    img {
+      width: 100%;
+      transform: scale(1);
```

Comment:
do we have a mixin for this image zoom style we use in a couple places? might be nice to add for consistency

Reply:
I looked into adding mixins but unfortunatly there are too many moving pieces and they all have different parent classes and hover triggers. However, the `scale()` on hover can be moved into a variable!